content/en/running-programs/_index.md

```diff
@@ -0,0 +1,57 @@
+
+## Considerations for Digital Health programs
+Below are the considerations made when running a digital health program:
+1. Scope - goals, deadlines, and program deliverables
+2. Technical requirements - devices , servers and storage requirements
+3. Ethics and regulations - governance,privacy and security which vary from country to country
+4. Landscape - general understanding of the region that the program will run in
+5. Healthcare ecosystem - standards, implementing partners, facilities and governing bodies
+6. Human resource requirements - skills to build and sustain the program
+
+## CHT Lifecycle
+CHT programs follow a process from beginning to end in order to deliver a complete functional solution. This process is referred to as the CHT Lifecycle. Although the process my differ from program to program, the process below is what has been used for most of our deployed programs.
+{{< figure src="cht_lifecycle.png" link="cht_lifecycle.png" class="center col-8 col-lg-8" >}}
+1. Initiation - introductions and initial conversation with the implementing partner. At this stage, the implementing partner defines the problem they would like to solve and the kind of program they would like to run. They also understand the various capabilities and features of the CHT.
+2. Planning - includes coming up with detailed workplans, schedules, budget forecasting and baseline data collection.
+3. Design - design plan sharing and reviewing, design field trips, definition of user personas, preperation of design workflows and program hierarchy.
+4. Building - refers to development of forms, tasks, targets, debugging and other aspects of CHT applications.
+5. Hosting - includes different hosting options, data migration, configuration and optimization of CHT applications in various environments.
+6. Implementation - refers to the process of putting CHT applications into action. It includes training plan development, logistics planning, expenditure forecasting and tracking, tech support training, application installation and end user training.
+7. Support - refers to activities done after deployment and implementation. It includes managing updates, building dashboards, monitoring and evaluation, impact measurement and user experience feedback.
+8. Closeout - includes budget tracking , discussions on scaling, report writing on impact and challenges experienced.
+
+## Program personas
+Various people interact with CHT programs at various stages.These people include program heads, program administrators,service designers,UX designers,software developers,system administrators and technical support officials.
+
+## Running successful community health programs
+Running a successful digital health program depends on various factors. Here are a couple of things we champion for:
+1. Community Health Workers compensation
+2. Privacy and protection of user data
+
```

Comment:
There are IMO heaps more to add here, and maybe worth having different sections, eg:

```
### Professional CHWs

<include the list from CHIC>

### Technical

- Privacy and protection of user data
- Monitoring
- Backups
- Redundancy
- Regular CHT upgrades
- Regular device maintenance (Android app and security updates)
- Regular device hardware replacement
- ...
```